app/Template/board/task_footer.php

```diff
@@ -1,6 +1,6 @@
 <?php if (! empty($task['category_id'])): ?>
-<div class="task-board-category-container">
-    <span class="task-board-category">
+<div class="task-board-category-container task-board-category-container-color">
+    <span class="task-board-category tag-<?= $this->text->e($task['category_name']) ?>">
```

Comment:
Using `category-` instead of `tag-` might be more appropriate because Kanboard have tags and categories?

Reply:
you are right, i changed the "tag-" value, also in the markdown informations. thanks for the information, i didnt notice this inconsistency!